dumpling/export/dump.go

```diff
@@ -1211,6 +1223,9 @@ func (d *Dumper) renewSelectTableRegionFuncForLowerTiDB(tctx *tcontext.Context)
 		tctx.L().Debug("no need to build region info because database is not TiDB 3.x")
 		return nil
 	}
+	d.selectTiDBTableRegionFunc = func(_ *tcontext.Context, _ *sql.Conn, meta TableMeta) (pkFields []string, pkVals [][]string, err error) {
```

Comment:
why set this func here? seems it have no work, `d.selectTiDBTableRegionFunc` will be set again in L1292

Reply:
Because for TiDB v3.0+, the original selectTiDBTableRegionFunc will always fail because TiDB v3.0 doesn't have `tidb_decode_key` function nor `DB_NAME`,`TABLE_NAME` columns in `INFORMATION_SCHEMA.TIKV_REGION_STATUS`. 
https://github.com/pingcap/tidb/blob/c497d5c/dumpling/export/dump.go#L775
To avoid this function continuously returning errors and confusing users because we fail to init this function at first, I put a function to return an ignorable function here.

I will add a comment here later.